File: kyw0716/level0/로그인 성공.js
function solution(id_pw, db) {
  const ids = [];
  const pwds = [];

  db.forEach((arr) => {
    ids.push(arr[0]);
    pwds.push(arr[1]);
  });

  if (ids.includes(id_pw[0])) {
    if (pwds[ids.indexOf(id_pw[0])] === id_pw[1]) return "login";
    return "wrong pw";
  }
  return "fail";
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120883

File: sk000801/입문/로그인 성공.java
import java.util.*;
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        int count = 0;
        String answer = "";
        for(int i = 0; i < db.length; i++) {
            if(id_pw[0].equals(db[i][0])) answer = id_pw[1].equals(db[i][1]) ? "login" : "wrong pw";
            else count++;
            
            if(answer.equals("login")) return answer;
            if(count == db.length) return "fail";
        }
        return "wrong pw";
    }
}